src/site/content/en/learn/forms/form-element/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Use forms to get data from users
+description: >
+  Learn the basics of using a form on the web with this introduction to the form element
+authors:
+  - michaelscharnagl
+date: 2021-09-06
+---
+
+Imagine you want to ask people on your website about their favorite animal.
+As a first step, you need a way to collect the data.
+
+How do you do this with HTML?
+
+{% Glitch 'learn-forms-get-started' %}
+
+In HTML, you can build this using the form element (`<form>`),
+an `<input>` with a `<label>`, and a submit `<button>`.
+
+{% Aside %}
+You may wonder where the styles for this example are coming from.
+They are coming from a general stylesheet
+[included in all demos](https://web.dev/learn/forms#demos).
+
+Interested in styling forms? You can learn about this in a later
+[module](/learn/forms/css/styling).
+{% endAside %}
+
+## What is a form element?
+
+```html
+<form>
+  <label for="animal">What is your favorite animal?</label>
+  <input type="text" id="animal" name="animal">
+  <button>Submit</button>
+</form>
+```
+
+The form element consists of the start tag `<form>`,
+optional attributes defined in the start tag, and an end tag `</form>`.
+
+Between the start and end tag, you can include form elements like `<ìnput>` and `<textarea>`
+for different types of user input.
+You will learn more about [form elements](/learn/forms/form-elements) in the next module.
+
+{% Aside %}
+Use HTTPS to protect all websites and forms,
+not only if you handle sensitive data. This way, all data is encrypted.
+
+Find out more: [Secure connections with HTTPS](/secure/#secure-connections-with-https).
+{% endAside %}
+
+## Where is the data processed?
+
+When the form is submitted (for example, when the user clicks the submit button),
+the browser makes a request.
+A script can respond to that request and process the data.
+
+{% Aside %}
+A script (running on the server or the client) is needed to process the form.
+It may [validate](/learn/forms/validation) the data, save it into a database,
+or do other operations based on the form data.
+{% endAside %}
+
+By default, the request is made to the page where the form is shown.
+
+Say you want a script running at `https://web.dev`
```

Comment:
I think we wouldn't normally put the URL in a code block.

Reply:
Hi @samthor  - I'm still working on this, was having issues locally so wanted to see if it would build online. I'll ping folk for a review once I'm ready :)

Reply:
in this case, the URLs are essentially part of the code (dealing with where a form submits to) so it seemed right to put them in code font, but I'll see what the others think.